### DISCUSSION TITLE
How can i get scripts from models in the workspace to show?

### DISCUSSION THREAD

**Discussion by RealMrCactus** [argon-rbx/argon#46] [Q&A]
self explanatory

**Comment by AridAjd** [comment]
If models in your workspace directory are `rbxm` files, then those files can just have the scripts and argon will port it over

**Comment by RealMrCactus** [comment]
> If models in your workspace directory are `rbxm` files, then those files can just have the scripts and argon will port it over

uhhhhh no? its just a model in studio

**Comment by AridAjd** [comment]
https://github.com/argon-rbx/argon/assets/118916772/dc99bf3e-6b7d-45b4-bd39-5e97f6a5e624

**Comment by DervexDev** [comment]
What do you mean by show? We have very little information on what you are trying to do.

**Comment by RealMrCactus** [comment]
> What do you mean by show? We have very little information on what you are trying to do.

In roblox studio i have a model with a script in it and i want to see that in vsc

**Comment by DervexDev** [comment]
Did you follow this [tutorial](https://argon.wiki/docs/getting-started/porting#porting-from-roblox-studio)? There is everything you need.

**Comment by DervexDev** [comment]
@RealMrCactus did it solve your problem?
